index.html

```diff
@@ -0,0 +1,150 @@
+                <div class="card"><img src="sources/mobile-version/images/superstar-icon.png" alt="" class="card-img">
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            <button class="button-cards" id="join">Join to the party!</button>
+            </div>
+            <div class="grid-container-desktop">
+                <div class="card"><img src="sources/mobile-version/images/mask-icon.png" alt="" class="card-img">
+                    <h2>thematic events</h2>
+                    <h3>Feel free to play as a character in the night of the cartoons, and roleplay in the other special events!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/tickets-icon.png" alt="" class="card-img">
+                    <h2>Free Bar & Entry</h2>
+                    <h3>You can ask for a free drink every day. As well get a free entry to one show in the last day of the event</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/superstar-icon.png" alt="" class="card-img">
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            </div>
+            <a href="" class="link-desktop">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="artists">
+            <div></div>
+            <h2>Featured Artists</h2>
+            <div class="tiny-line"></div>
+            <div class="container-speakers">
+            </div>
```

Comment:
- [ ] please, like the previous reviewer noted, the "Featured speakers" section has a 2-column layout with 3 speakers on each column. You have just 4 speakers now in your desktop version. Remember, we are reproducing someone's design and this is an opportunity for you to learn how to recreate something while maintaining the form but changing the content.
- [ ] Great job adding the gray pattern background on each speaker image like the previous reviewer requested. please kindly fit them well behind the images. 
yours

![Screenshot (1171)_LI](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/e868e6bb-36c0-4ba5-b21d-c1a2aefef610)
original
![Screenshot (1173)](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/445a4c04-ad4b-4635-9bd1-59a823e4c8f3)